config.py

```diff
@@ -7,14 +7,15 @@ class Config(object):
     PORT = os.getenv('PORT', 8085)
     LOGGING_LEVEL = os.getenv('LOGGING_LEVEL', 'INFO')
     USE_SESSION_FOR_NEXT = True
-
-    BACKSTAGE_API_URL = os.getenv('BACKSTAGE_API_URL', 'http://localhost:8001/backstage-api')
     RESPONSE_OPERATIONS_UI_SECRET = os.getenv('RESPONSE_OPERATIONS_UI_SECRET', "secret")
+    BACKSTAGE_BASE_URL = os.getenv('BACKSTAGE_BASE_URL')
+    BACKSTAGE_API_SEND = '/v1/secure-message/send-message'
```

Comment:
I still don't like this.  Can anyone else weigh in? @JamesGardiner @warrenbailey?  The endpoint changes so infrequently that it's just adding config for the heck of it.  If we were to go in this direction we'd have to do it for every single endpoint that we point to in backstage which would massively bloat the config file for no reason

Reply:
Agreed; we have not long completed an exercise to reduce our config overhead. Config should be exclusively for things that change between environments or things that change frequently.

Reply:
I disagree totaly. Everybody knows that String is Evil , and when you test something , you do not want to test a String but a configuration . Why do you want to change Strings over a Constant based on your running porfile .